CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 -----
 
+## `v2.1.0-beta`
+
+- Uses go-autorest v7.0.2 (Polling logic changed).
+- Cancel channel argument added in long-running calls to cancel polling if needed.
+- Allow caller to provide headers for DeleteBlob methods.
```

Comment:
can you please group these under `storage`? it's vague that where they belong. In the future you can also provide better short descriptions. Take a look at this:

```
- storage: Allow caller to provide headers for DeleteBlob methods.
- storage: Enables connection sharing with http keepalive
```